adopt-github-release/src/main/groovy/net/adoptium/release/UploadFiles.groovy

```diff
@@ -36,6 +36,7 @@ class UploadAdoptReleaseFiles {
                 // Only release file names containing "hotspot" or "sources"
```

Comment:
Pre-existing, but this comment seems outdated. Thoughts?

Reply:
I had a change to that in an earlier version but it got lost. Good spot. I've updated to make it more generic and indicate that it's intended to ensure that we don't publish non-temurin builds.

Reply:
Thanks!